libevmasm/Assembly.cpp

```diff
@@ -186,11 +186,10 @@ class Functionalizer
 
 	void printLocation()
 	{
-		if (!m_location.sourceName && m_location.isEmpty())
+		if (!m_location.source && m_location.isEmpty())
```

Comment:
Shouldn't this be `||`?

Reply:
actually yes. I feel like i'm not concentrated, sorry.